tests/scripts/keadm_deprecated_e2e.sh

```diff
@@ -120,14 +77,11 @@ echo -e "\nUsing latest commit code to do keadm_deprecated_e2e test..."
 echo -e "\nBuilding keadm..."
 build_keadm
 
-export KUBECONFIG=$HOME/.kube/config
-
 echo -e "\nPreparing cluster..."
-prepare_cluster
+util::prepare_cluster
 
 install_cni_plugins
 
-kubeedge_version=${VERSION: 1}
```

Comment:
The original rationale does not appear to enable the ability to test different versions of kubeedge.
Is your change affecting the original logic?